docs/source/column.rst

```diff
@@ -40,6 +40,7 @@ Column class reference
 .. autosummary::
     :toctree: generated
     :nosignatures:
+    :template: class.rst
```

Comment:
You may want to check if the output page looks as you expect.

Reply:
yep it looks the same to me with or without haha.